addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -163,29 +230,75 @@
+            <field name="name">19% Einfuhrumsatzsteuer (abzugsf. Vorsteuer)</field>
             <field name="description">19% EUSt</field>
             <field name="amount_type">percent</field>
             <field name="amount">19</field>
             <field name="refund_account_id" ref="account_1588"/>
             <field name="account_id" ref="account_1588"/>
-            <field name="type_tax_use">purchase</field>
+            <field name="type_tax_use">none</field>
             <field name="tag_ids" eval="[(6,0,[ref('l10n_de.tag_de_62')])]"/>
             <field name="tax_group_id" ref="tax_group_19"/>
         </record>
-        <record id="tax_import_7_skr03" model="account.tax.template">
+        <record id="tax_import_19_payable_skr03" model="account.tax.template">
+            <field name="sequence">98</field>
+            <field name="chart_template_id" ref="l10n_de_chart_template"/>
+            <field name="name">19% Einfuhrumsatzsteuer (zu bezahlender Aufschub)</field>
+            <field name="description">19% EUSt (zu bezahlender Aufschub)</field>
+            <field name="amount_type">percent</field>
+            <field name="amount">-19</field>
+            <field name="refund_account_id" ref="account_1788"/>
+            <field name="account_id" ref="account_1788"/>
+            <field name="type_tax_use">none</field>
+            <field name="tax_group_id" ref="tax_group_19"/>
+        </record>
+        <record id="tax_import_19_and_payable_skr03" model="account.tax.template">
             <field name="sequence">21</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">7% Einfuhrumsatzsteuer</field>
-            <field name="description">7% EUSt</field>
+            <field name="name">19% Einfuhrumsatzsteuer (§21 Abs.3 UstG)</field>
+            <field name="description">Einfuhrumsatzsteuer 19%</field>
+            <field name="amount_type">group</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+            <field name="children_tax_ids" eval="[(6,0,[ref('tax_import_19_skr03'), ref('tax_import_19_payable_skr03')])]"/>
+            <field name="tax_group_id" ref="tax_group_19"/>
+        </record>
+        <record id="tax_import_7_skr03" model="account.tax.template">
+            <field name="sequence">98</field>
+            <field name="chart_template_id" ref="l10n_de_chart_template"/>
+            <field name="name">7% Einfuhrumsatzsteuer (abzugsf. Vorsteuer)</field>
+            <field name="description">7% EUSt (abzugsf. Vorsteuer)</field>
             <field name="amount_type">percent</field>
             <field name="amount">7</field>
             <field name="refund_account_id" ref="account_1588"/>
             <field name="account_id" ref="account_1588"/>
-            <field name="type_tax_use">purchase</field>
+            <field name="type_tax_use">none</field>
```

Comment:
import tax is not purchase?  Where is it applied then?

Reply:
It is now a children of a new parent tax (starting from line 291). The parent is a purchase tax use.